cmd/client.go

```diff
@@ -25,106 +24,134 @@ import (
+func newCliCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "cli",
+		Short: "Manage replication task and TiCDC cluster",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			etcdCli, err := clientv3.New(clientv3.Config{
+				Endpoints:   []string{cliPdAddr},
+				DialTimeout: 5 * time.Second,
+				DialOptions: []grpc.DialOption{
+					grpc.WithConnectParams(grpc.ConnectParams{
+						Backoff: backoff.Config{
+							BaseDelay:  time.Second,
+							Multiplier: 1.1,
+							Jitter:     0.1,
+							MaxDelay:   3 * time.Second,
+						},
+						MinConnectTimeout: 3 * time.Second,
+					}),
+				},
+			})
 			if err != nil {
 				return err
 			}
-			startTs = oracle.ComposeTS(ts, logical)
-		}
-		err = verifyStartTs(ctx, startTs, cli)
-		if err != nil {
-			return err
-		}
+			cdcEtcdCli = kv.NewCDCEtcdClient(etcdCli)
+			pdCli, err = pd.NewClient([]string{cliPdAddr}, pd.SecurityOption{})
+			if err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+	command.AddCommand(
+		newCreateChangefeedCommand(),
+		newListCaptureCommand(),
+		newListChangefeedCommand(),
+		newListProcessorCommand(),
+		newQueryChangefeedCommand(),
+		newQueryProcessorCommand(),
+		newGetTsoCommand(),
+		newTruncateCommand(),
+	)
+	command.PersistentFlags().StringVar(&cliPdAddr, "pd", "http://127.0.0.1:2379", "PD address")
+
+	return command
+}
 
-		cfg := new(util.ReplicaConfig)
-		if len(configFile) > 0 {
-			if err := strictDecodeFile(configFile, "cdc", cfg); err != nil {
+func newCreateChangefeedCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "create",
+		Short: "create a new replication task (changefeed)",
+		Long:  ``,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			id := uuid.New().String()
+			if startTs == 0 {
+				ts, logical, err := pdCli.GetTS(ctx)
+				if err != nil {
+					return err
+				}
+				startTs = oracle.ComposeTS(ts, logical)
+			}
+			if err := verifyStartTs(ctx, startTs, cdcEtcdCli); err != nil {
 				return err
 			}
-		}
 
-		detail := &model.ChangeFeedInfo{
-			SinkURI:    sinkURI,
-			Opts:       make(map[string]string),
-			CreateTime: time.Now(),
-			StartTs:    startTs,
-			TargetTs:   targetTs,
-			Config:     cfg,
-		}
+			cfg := new(util.ReplicaConfig)
+			if len(configFile) > 0 {
+				if err := strictDecodeFile(configFile, "cdc", cfg); err != nil {
+					return err
+				}
+			}
 
-		for _, opt := range opts {
-			s := strings.Split(opt, "=")
-			if len(s) <= 0 || len(s) > 2 {
-				fmt.Printf("omit opt: %s", opt)
+			info := &model.ChangeFeedInfo{
+				SinkURI:    sinkURI,
+				Opts:       make(map[string]string),
+				CreateTime: time.Now(),
+				StartTs:    startTs,
+				TargetTs:   targetTs,
+				Config:     cfg,
 			}
 
-			var key string
-			var value string
+			for _, opt := range opts {
+				s := strings.Split(opt, "=")
+				if len(s) <= 0 || len(s) > 2 {
+					fmt.Printf("omit opt: %s", opt)
```

Comment:
should `break` here?

Reply:
If we want to omit the invalid option, a `continue` should be here.